q2_taxa/assets/barplot/src/toolbar.js

```diff
@@ -163,6 +163,13 @@ export function addSortByPicker(row, svg, data, dataMeta) {
       const sel = _appendSortByPicker(grp, svg, data, dataMeta);
       _sortBySelectOptions(sel, metaData, sortedKeysReverse, sortedKeysReverse[0]);
     });
+  grp.append('button').text('-')
```

Comment:
**(NOTE: Some of the line-links in the below comment are in the folded portions of the diff. By default GH won't expand those.)**

Move this whole thing to the [order select level](https://github.com/qiime2/q2-taxa/pull/46/files#diff-735c7160e4fd341ec6d00259c0e7b27eR42), that way we get a (-) on every selector.

Probably will need to add a new column to the group, and a boolean arg (`delete`), which controls whether or not the delete button is drawn:

```javascript
function _appendSortByPicker(sel, svg, data, dataMeta, delete) {
  const row = sel.append('div').attr('class', 'row');
  const colA = row.append('div').attr('class', 'col-lg-1');
  const colB = row.append('div').attr('class', 'col-lg-4');
  const colC = row.append('div').attr('class', 'col-lg-3');
  const colD = row.append('div').attr('class', 'col-lg-4');

  if (delete) {
    colA.append('input')
      .attr('type', 'button')
      .attr('value', 'Delete')
      .on('click', () => { row.remove(); _updateSort(sel, svg, data, dataMeta); });
  }
...
```

Probably makes sense to set `delete` to `true` [here](https://github.com/qiime2/q2-taxa/pull/46/files#diff-735c7160e4fd341ec6d00259c0e7b27eR163), and to `false` [here](https://github.com/qiime2/q2-taxa/pull/46/files#diff-735c7160e4fd341ec6d00259c0e7b27eR174). This makes it so that there is at least one sorter, which means we don't have to handle the edge case where there is none.